src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -66,7 +66,16 @@ Note that if you're on our Standard tier and want to assign a user to a [default
   A diagram explaining how you can grant user groups access to roles and accounts. Note that this applies to users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (and not our original user model).
 </figcaption>
 
-Here are more explanations of our user management terms and how they relate to each other:
+Some tips on setting up access grants: 
+
+* Before adding users, it can help to plan out how your New Relic organization will be structured. How many accounts will you have? What user groups will get access to what accounts? Will you use our default groups and roles or create your own custom groups and roles? It can help to use a spreadsheet to map out your organization and how you'll enable access grants. 
+* For smaller, flatter organizations okay with full internal transparency, you may only need a couple groups. 
+* A common configuration for organizations with 20 or more accounts is setting up groups with the more organization-scoped roles (**Organization manager**, **Authentication domain manager**, and **Billing user**) on the primary account, and then on other accounts, setting up groups with the more product-scoped roles (like **All product admin**, **Standard user**, or custom roles). 
```

Comment:
Long sentence, split in two maybe? "... primary account. Then, on other accounts, set up groups..."